packages/cli/test/fixtures/build-modules/expected/bundled-amd/shared_bundle_1.js

```diff
@@ -19,8 +19,12 @@ define(["exports"], function (_exports) {
   _exports.$mod2 = mod2$1;
   const mod1 = 'mod1';
   _exports.mod1 = mod1;
+  sideEffectFunction();
+  /*@__PURE__*/
```

Comment:
What is @__PURE__ ?

Reply:
This is an annotation that was introduced initially by UglifyJS to mark "pure" functions (having no side effects) so they can be safely removed unless their result is actually used.

Reply:
Rollup honors it and does not remove function calls unless it is certain they have no side effect.